tests/phpunit/integration/tests/Admin/Customizer.php

```diff
@@ -34,12 +35,12 @@ class Customizer extends DependencyInjectedTestCase {
 	/**
 	 * Test instance.
 	 */
-	private \Google\Web_Stories\Admin\Customizer $instance;
+	private TheCustomizer $instance;
 
 	/**
 	 * Test instance.
 	 */
-	private \WP_Customize_Manager $wp_customize_mock;
+	private ?WP_Customize_Manager $wp_customize_mock = null;
```

Comment:
Since this is not a mock anymore, perhaps we should rename this. Maybe to `wp_customize_manager`?

Reply:
Done